warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -919,7 +919,7 @@ public void run() {
                 
                 // import the directory
                 log.info("Bringing Map Files online for " + tableName);
-                accumuloClient.tableOperations().importDirectory(tableName, tableDir.toString(), failuresDir, false);
+                accumuloClient.tableOperations().importDirectory(tableDir.getName()).to(tableName).tableTime(false);
```

Comment:
the directory used to be tableDir, but now it's tableDir.getName.  Is it ok not to get the full URI?  also, is there no failure directory in 2.x?

Reply:
toString() is probably correct, based on looking at some accumulo tests.
FWIW, the docs here https://accumulo.apache.org/release/accumulo-2.0.0/ say:

- Inspection of file on the client side. Inspection of all files is done before the FATE operation starts. This results in less namenode operations and fail-fast for bad files (no longer need a fail directory).

Specifying a failureDir will be gone when the current deprecated code is removed (3.0+).

Reply:
got it.  thanks for letting me know.